samples/native_training/native_training.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import argparse
+import os
+
+import functorch
+from functorch._src.compile_utils import strip_overloads
+import iree_torch
+import numpy as np
+from sklearn.datasets import make_regression
+from sklearn.model_selection import train_test_split
+import torch
+import torch_mlir
+
+# Dataset
+X, y, coef = make_regression(n_features=3, coef=True)
+X = torch.from_numpy(X).to(dtype=torch.float32)
+y = torch.from_numpy(y).to(dtype=torch.float32)
+X, X_test, y, y_test = train_test_split(X, y)
```

Comment:
as a matter of understandability, can we write this without sklearn and numpy? Most people using PyTorch probably are not familiar with sklearn and I don't think there is anything here specific to numpy. I think make_regression is as simple as (coeff * torch.ones(...)) + bias + torch.randn(...), and test_train_split can be done with a simple indexing operation.

Reply:
Good call.  This is a bit of a leftover from when I started bootstrapping the model and making sure it worked, but the model itself certainly no longer needs this.  I've replaced it with simple dummy values.